src/main/java/com/yevhenberladyniuk/debts/service/PartnerService.java

```diff
@@ -0,0 +1,21 @@
+package com.yevhenberladyniuk.debts.service;
+
+import com.yevhenberladyniuk.debts.domain.Partner;
+import com.yevhenberladyniuk.debts.dto.CreatePartnerForm;
+import com.yevhenberladyniuk.debts.dto.PartnerDto;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface PartnerService {
+
+    void create(CreatePartnerForm createPartnerForm, Long userId);
+
+    List<Partner> findAllByUserId(Long userId);
+
+    void deletePartnerById(Long id);
+
+    Optional<Partner> findPartnerById(Long id);
```

Comment:
method should receive `user`, cause now any user can receive any partner by id
Each user is allowed only to find partners which they created